rwengine/src/audio/ReverbEffect.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef REVERBEFFECT_H
+#define REVERBEFFECT_H
+
+#include "SoundEffect.hpp"
+
+class ReverbEffect : public SoundEffect {
+
+public:
+    ReverbEffect();
+
+    void	setDensity             ( float density = 1.0f   );
```

Comment:
Remove table-like formatting?

Reply:
I think there should be some comments describing parameters that functions are setting